setup.py

```diff
@@ -0,0 +1,3 @@
+from setuptools import setup
+
+setup(use_scm_version=True,)
```

Comment:
This trailing comma looks weird when the call is a single line. Is this a convention from somewhere that I'm not familiar with? (I'm a bit surprised black didn't remove it.)

Reply:
It seems that black got halfway through reformatting, but didn't drop the trailing comma. It accepts the fixed form.